x/inter-tx/types/msgs.go

```diff
@@ -34,10 +33,12 @@ func (MsgRegisterAccount) Type() string {
 	return TypeMsgRegisterAccount
 }
 
-// ValidateBasic performs a basic check of the MsgRegisterAccount fields.
 func (msg MsgRegisterAccount) ValidateBasic() error {
-	return nil
+	if msg.Owner == "" {
+		return sdkerrors.Wrap(sdkerrors.ErrInvalidAddress, "missing sender address")
+	}
 
+	return nil
 }
 
 func (msg MsgRegisterAccount) GetSignBytes() []byte {
```

Comment:
It is good to be aware that not supporting amino means no ledger support in the short term future. Ledger support for proto messages is still TBD

Reply:
Interesting. Thanks for letting me know. I think for now this is ok but may be something to think on generally.